kernel_gateway/gatewayapp.py

```diff
@@ -152,9 +153,16 @@ def _default_kernel_name_default(self):
     def _list_kernels_default(self):
         return os.getenv(self.list_kernels_env, 'False') == 'True'
 
+    api_env = 'KG_API'
+    api = Unicode(config=True,
+        help='Designates that the kernel gateway will serve APIs from this notebook (KG_API env var)'
```

Comment:
Do we need a dedicated option for that plus another specifying the notebook whose APIs should be exposed, or just renaming and documenting this one for clarity?